contracts/erc20-bridge-sampler/contracts/src/ERC20BridgeSampler.sol

```diff
@@ -512,6 +525,26 @@ contract ERC20BridgeSampler is
         }
     }
 
+    /// @dev Returns the address of a liquidity provider for the given market
+    ///      (takerToken, makerToken), from a registry of liquidity providers.
+    /// @param takerToken Taker asset managed by liquidity provider.
+    /// @param makerToken Maker asset managed by liquidity provider.
+    /// @return Address of the liquidity provider.
+    function getLiquidityProviderFromRegistry(
+        address registryAddress,
+        address takerToken,
+        address makerToken
+    )
+        public
+        view
+        returns (address)
+    {
+        return ILiquidityProviderRegistry(registryAddress).getLiquidityProviderForMarket(
```

Comment:
This method [may revert](https://github.com/0xProject/plp/blob/development/contracts/src/PLPRegistry.sol#L75) if a liquidity pool does not exist. How do we handle reverts?